### ISSUE TITLE
Table sorting broken on customer orders view in admin

### ISSUE THREAD

**Issue by alexrowesoap** [Sylius/Sylius#10551]
**Sylius version affected**: 1.5

**Description**  
The table header has the icons and anchors indicating it is sortable, but clicking the headers yields no sorting actions

**Steps to reproduce**  
- Log into the admin
- Find a customer that has two or more orders on /admin/customers/
- Click the "Show orders" button
- Click on the headers of the table to action the sort, and observe it not sorting

**Comment by tom10271** [comment]
I am not sure but it might be caused by broken JS

```
app.js:1 Uncaught TypeError: Cannot read property 'querySelectorAll' of null
    at Os.value (app.js:1)
    at new Os (app.js:1)
    at HTMLDocument.<anonymous> (app.js:1)
    at e (app.js:1)
    at t (app.js:1)
```

**Comment by pamil** [comment]
Fixed by #10583.
